docs/locales/ko_KR/LC_MESSAGES/common-api.po

```diff
@@ -59,7 +59,7 @@ msgid ""
 "For local deployments, you may create a master dummy pair in the "
 "configuration (TODO)."
 msgstr ""
-"로컬 배포의 경우, 환경 설정에 마스터 더미 쌍을 만들 수 있습니다. (TODO)"
+"로컬 배포의 경우, 환경 설정에 완전히 무수한 복사본을 만들 수 있습니다. (TODO)"
```

Comment:
Dummy has the meaning of 'imitation', so each dummy pair may have different content. Therefore, it would be better to translate it into a meaning other than a copy so that the user does not misunderstand.